surface/include/pcl/surface/3rdparty/poisson4/bspline_data.hpp

```diff
@@ -446,7 +446,7 @@ namespace pcl
     {
       POISSON_THROW_EXCEPTION (pcl::poisson::PoissonBadArgumentException, "B-spline up-sampling not supported for degree " << Degree);
     }
-    template<>
+    template<> inline
```

Comment:
Create a cpp file for the inline additions in this file as well. The corresponding CMake file is in
https://github.com/PointCloudLibrary/pcl/blob/f82719543bdb2848ce807bb58d9a98ead1eab5da/surface/CMakeLists.txt#L84-L88

Reply:
Didn't see that. Modified.